### PR TITLE
Improve the servers tutorial

### PR DESCRIPTION
**Pull request by BalaM314**
Makes it more accessible to new players and users who are not familiar with command prompt.

### CODE REVIEW

File: docs/servers.md [hunk 1]
```diff
@@ -10,12 +10,14 @@ You can connect to one using the "Join Game" button under the "Play" menu. Unlik
 
 To establish a dedicated server, a dedicated Linux or Windows machine is **highly** recommended.
 
-1. If you haven't already, install at least JRE and JDK 8.
```
Comment (BalaM314) [inline docs/servers.md:13]: JDK is only necessary if you want to build the server jar from source. If you know how to do that, you don't probably don't need to be told to install it.

File: docs/servers.md [hunk 2]
```diff
@@ -10,12 +10,14 @@ You can connect to one using the "Join Game" button under the "Play" menu. Unlik
...
-2. Download the desired server release from [itch.io](https://anuke.itch.io/mindustry), or compile one yourself. 
-3. Open a terminal or TTY session then change `cd` to the directory the JAR is placed in.
-4. Run `java -jar server.jar` using Command Prompt (on Windows) or your favorite terminal (on Linux and Mac). The commands are explained in the `help` command.
-5. Start hosting a map with `host <mapname> [mode]` after you configured your server.
-6. If you are using Windows to run your server, use your favorite search engine to look up how to add rules to your Windows Firewall, as it blocks that port most of the time. Make sure to allow **port 6567 TCP and UDP**.
+1. If you haven't already, install Java 8. (If you can launch the client, this step is already done.)
+2. Download the desired server release from [itch.io](https://anuke.itch.io/mindustry), [Github releases](https://github.com/Anuken/Mindustry/releases). The filename should be `server-release.jar`, although if you are on Windows it may only show as `server-release`.
```
Comment (BalaM314) [inline docs/servers.md:14]: The file not showing the .jar extension in Windows confused some players

File: docs/servers.md [hunk 3]
```diff
@@ -10,12 +10,14 @@ You can connect to one using the "Join Game" button under the "Play" menu. Unlik
...
+3. Move the server-release file to your desired folder. It is recommended to place it in a new directory on your desktop.
+4. Open a new terminal window(CMD if you are on windows). Use the command "[cd](https://www.digitalcitizen.life/command-prompt-how-use-basic-commands/)" to navigate into the directory you placed the server-release file in, ex `cd Desktop` then `cd Server`. Run `java -jar server-release.jar`.
```
Comment (BalaM314) [inline docs/servers.md:16]: Try to explain CD and link to external article, correct server.jar to server-release.jar

File: docs/servers.md [hunk 4]
```diff
@@ -10,12 +10,14 @@ You can connect to one using the "Join Game" button under the "Play" menu. Unlik
...
+1. If you haven't already, install Java 8. (If you can launch the client, this step is already done.)
+2. Download the desired server release from [itch.io](https://anuke.itch.io/mindustry), [Github releases](https://github.com/Anuken/Mindustry/releases). The filename should be `server-release.jar`, although if you are on Windows it may only show as `server-release`.
+3. Move the server-release file to your desired folder. It is recommended to place it in a new directory on your desktop.
+4. Open a new terminal window(CMD if you are on windows). Use the command "[cd](https://www.digitalcitizen.life/command-prompt-how-use-basic-commands/)" to navigate into the directory you placed the server-release file in, ex `cd Desktop` then `cd Server`. Run `java -jar server-release.jar`. 
+5. Your server should have started. Type `help` for a list of all console commands. If you get a message similar to "Unable to access file server-release.jar" you are in the wrong directory.
```
Comment (BalaM314) [inline docs/servers.md:17]: Add explanation for most common error

**Comment by BalaM314** [comment]
(still does not know how to use GH reviews)

File: docs/servers.md [hunk 5]
```diff
@@ -10,12 +10,14 @@ You can connect to one using the "Join Game" button under the "Play" menu. Unlik
 
 To establish a dedicated server, a dedicated Linux or Windows machine is **highly** recommended.
 
-1. If you haven't already, install at least JRE and JDK 8.
-2. Download the desired server release from [itch.io](https://anuke.itch.io/mindustry), or compile one yourself. 
-3. Open a terminal or TTY session then change `cd` to the directory the JAR is placed in.
-4. Run `java -jar server.jar` using Command Prompt (on Windows) or your favorite terminal (on Linux and Mac). The commands are explained in the `help` command.
-5. Start hosting a map with `host <mapname> [mode]` after you configured your server.
-6. If you are using Windows to run your server, use your favorite search engine to look up how to add rules to your Windows Firewall, as it blocks that port most of the time. Make sure to allow **port 6567 TCP and UDP**.
+1. If you haven't already, install Java 8. (If you can launch the client, this step is already done.)
```
Comment (Anuken) [inline docs/servers.md:13]: - The client comes with Java installed, so the part in parentheses isn't true.
- Java 8 is very old; I'm not sure if it's even available everywhere now. People should just be told to install Java instead.

File: docs/servers.md (hunk 2 above)
Comment (Anuken) [inline docs/servers.md:14]: It's not clear which version you want them to download here. The filename is `server.jar` on itch once extracted, not `server-release.jar`. The old guide only links to itch, so mentioning "server-release" and linking both here is confusing.

**Review by Anuken — CHANGES_REQUESTED** [review]

File: docs/servers.md (hunk 2 above)
Comment (BalaM314) [reply docs/servers.md:14]: Thanks, didn't notice that.

**Pull request merged**